### DISCUSSION TITLE
Let's all use our heads….

### DISCUSSION THREAD

**Discussion by mavedaxwell** [phcode-dev/phoenix#1887] [General]
Can we collectively not put popup survey panels that can't be closed in the interface, or at least wait until the user has, say like, 6 hours of time on the app. I wish there was someone here who could appreciate the idea that a survey that pops up during what is work is high level idiocy. Tell the marketers that the product is the platform not their marketing whims. And yeah, I know its a free and beta (Phoenix Code) but for good god sake use your heads for one second… 

I don't trust how your platform works. I had about 30 minutes of time until the pop up survey came up. Uncloseable and on top of my work. Im more than happy to provide feedback for ideas and products to get better but common sense is not present in your choice here and now I call into question the sense that was used to build this thing.

**Comment by charlypa** [comment]
@mavedaxwell 
Thank you for sharing your feedback. We understand your concerns regarding the timing and implementation of surveys within the platform. We'll definitely take this into account as we continue to refine the user experience. Your input is valuable to us, and we appreciate your patience as we work to improve the product.

**Comment by abose** [comment]
@mavedaxwell Te survey can be closed by clicking on the close button as seen in the below image. it is not mandatory to fill. its disposable instantly. Were you not seeing an option to close the survey dialog? 

The survey dialog helped us reach this far as we almost 95% of the time implement features suggested in the survey for the next release. A lot of UX issues were addressed with user reports from surveys from the early days of Phoenix beta and it stayed as it was doing its purpose to help users very well.

If the close button doesn't work for you or if it can be done in a better way, Please comment here so that we can fix the issue.

![image](https://github.com/user-attachments/assets/7d71fcb0-0495-4c8e-b7c6-816bd084b474)

**Comment by Eshku** [comment]
Yea, I agree, popups shouldn't be a thing in IDE at all.
Most of the times I open small IDE like brackets \ phoenix \ sublime - I'm probably there to do quick and light work, edit few things and close it, but popups interrupts workflow.

Just add a button somewhere where it won't interrupt and won't be annoying.
Like idk, after "help" - put a button "Help is improve" or something like that, which will open current survey.
Same goes for popup on the launch, it sucks, it shouldn't be there by default.

That was actually main reason I decided to just stick with brackets for now instead of upgrading to phoenix.
If they do same thing, but one with popups, choice is obvious.
(Another reason is I'm not ready to commit to configuring it, but that's another story)

_____
As a little bit off top to survey itself as feedback:
There is one big thing that makes old-fashion IDE's absolute compared to VSCode for actual work - it's lack of AI.
We are at the point where AI is still not very smart at complex work (bad at actual coding complex things), but still can be used for simple tasks, help with manual work, give some decent suggestions, can be used right there in the IDE instead of googling in the browser...
I'm not sure if it makes sense to improve IDE itself until there is an AI integration. Gemini, Copilot - as paid options , codeium, phind - free.
There's probably more at this point, either way it should have something to be able to compete.
